src/ui/components/pages/importEmail/signContext.tsx

```diff
@@ -0,0 +1,161 @@
+import clsx from 'clsx';
```

Comment:
I think this file should be moved into a more generic place, like `messages` folder may be or `_core`. `signWrapper` was here because we initially implemented `email` and `ledger` on top of each other and forgot to move it out of here :-)